packages/build-tools/src/steps/functions/calculateEASUpdateRuntimeVersion.ts

```diff
@@ -76,7 +76,9 @@ export function calculateEASUpdateRuntimeVersionFunction(): BuildFunction {
         env,
       });
       if (resolvedRuntimeVersion) {
-        outputs.resolved_eas_update_runtime_version.set(resolvedRuntimeVersion);
+        outputs.resolved_eas_update_runtime_version.set(
+          resolvedRuntimeVersion.runtimeVersion ?? undefined
+        );
```

Comment:
This function will probably need to have a second output `resolved_eas_update_fingerprint_sources` so we do exactly the same thing for custom builds as for standard builds

Reply:
The fingerprints are now stored in a GC bucket.
I think this means that the fingerprint diffing won't work on a local build because you won't be able to access the signed url (due to cors?).
I imagine it's ok to have this as an EAS cloud only feature though.

Reply:
Yeah, that's true, in its current form, it wouldn't work for local builds because there would be no place to generate the signed URL for the fingerprint to be downloaded (local builds don't go through Turtle and WWW). 

Local builds use something called local-build-plugin
https://github.com/expo/eas-build/blob/69bbd3e7490b75a422e7439567d9627914d851a4/packages/local-build-plugin/src/main.ts#L1-L23

You can add support for it by basically not uploading the fingerprint to GCS when running the local build but copying it to some temp dir locally and then getting it from there during the build process.

That's what we do for project archive sources.

This will require supporting `PATH` source for fingerprint.